tests/drop_counters/test_drop_counters.py

```diff
@@ -247,17 +323,21 @@ def base_verification(discard_group, pkt, ptfadapter, duthost, combined_counter,
         pytest.fail("Incorrect 'discard_group' specified. Supported values: 'L2' or 'L3'")
 
 
-def test_equal_smac_dmac_drop(ptfadapter, duthost, setup, tx_dut_ports, pkt_fields):
+def test_equal_smac_dmac_drop(ptfadapter, fanout, duthost, setup, tx_dut_ports, pkt_fields):
     """
     @summary: Verify that packet with equal SMAC and DMAC is dropped and L2 drop cunter incremented
     """
     dut_iface, ptf_tx_port_id, dst_mac, src_mac = get_test_ports_info(ptfadapter, duthost, setup, tx_dut_ports)
+    src_mac = "00:00:00:00:00:11"
```

Comment:
This means we assume other fanouts use the same approach with openflow rules, while they can simply enable passthrough for SMAC==DMAC packets.

Reply:
Changed approach, now added possibility to define vendor specific module and use it inside 'fanouthost' fixture and to update/restore config.

Reply:
No, I mean what strange is that the test has to send smac==dmac traffic, however the test sends different smac.
I understand that the motivation on mellanox fanout is to add this openflow rule, however another platform may just enable passthrough for smac==dmac packets.

Reply:
Fixed. Now default behavior is that dmac assigns to smac (smac=smac).
If there is a need for specific vendor they can change behavior.
Here in Mellanox case - fanout configuration and smac value are updated, for other vendors the behavior is default.